template/src/ontology/Makefile.jinja2

```diff
@@ -426,22 +431,41 @@ KEEPRELATIONS=keeprelations.txt
 {% endif -%}
 
 CLEANFILES=$(MAIN_FILES) $(SRCMERGED) $(EDIT_PREPROCESSED)
+
 # This should be executed by the release manager whenever time comes to make a release.
 # It will ensure that all assets/files are fresh, and will copy to release folder
-
 .PHONY: prepare_release
-prepare_release: all_odk
-	rsync -R $(RELEASE_ASSETS) $(RELEASEDIR) &&\{% if project.sssom_mappingset_group is defined %}{% if project.sssom_mappingset_group.released_products is defined %}
-	mkdir -p $(RELEASEDIR)/mappings && cp -rf $(RELEASED_MAPPING_FILES) $(RELEASEDIR)/mappings &&\{% endif %}{% endif %}{% if project.use_dosdps %}
-	mkdir -p $(RELEASEDIR)/patterns && cp -rf $(PATTERN_RELEASE_FILES) $(RELEASEDIR)/patterns &&\{% endif %}
-	rm -f $(CLEANFILES) &&\
-	echo "Release files are now in $(RELEASEDIR) - now you should commit, push and make a release \
+prepare_release: all_odk copy_release_files
```

Comment:
Minor nitpick: since copy_release_files does not depend on anything, a make processor that parallelises goal execution would end up copying files that have not been updated yet. I am not too bothered, as I am sure that problem exists of 100 more things in the makefile, but I personally would have done:

```
prepare_release: all_odk
    $(MAKE) copy_release_files
```

No objections though to leave as is.

Reply:
> a make processor that parallelises goal execution would end up copying files that have not been updated yet

😱 This is a _very_ good point and definitely not a “minor nitpick”. I don’t think we officially support parallel builds (I know _I_ never tried one), and I agree with your assessment that there would probably be many problems with them). Still, that doesn’t mean that we should deliberately accept something that gets in the way of parallel builds.

> personally I would have done:
> ```
> prepare_release: all_odk
>    $(MAKE) copy_release_files 
> ```

Agreed!